lib/core/config.js

```diff
@@ -222,8 +232,29 @@ Config.prototype.loadContractsConfigFile = function() {
 
   let configFilePath = this._getFileOrOject(this.configDir, 'contracts', 'contracts');
 
-  const newContractsConfig = this._mergeConfig(configFilePath, configObject, this.env);
-
+  let newContractsConfig = this._mergeConfig(configFilePath, configObject, this.env);
+  if (newContractsConfig.gas != "auto") {
+      newContractsConfig.gas = utils.getWeiBalanceFromString(newContractsConfig.gas, web3);
+  }
+  if ('deployment' in newContractsConfig) {
+    if ('accounts' in newContractsConfig.deployment) {
+      for (var i = 0; i < newContractsConfig.deployment.accounts.length; i++) {
+        newContractsConfig.deployment.accounts[i].balance = utils.getWeiBalanceFromString(newContractsConfig.deployment.accounts[i].balance, web3);
+      }
+    }
+  }
+  for (var contractName in newContractsConfig.contracts) {
+    if (newContractsConfig.contracts[contractName].gas != "auto") {
+      if (newContractsConfig.contracts[contractName].gas) {
```

Comment:
These two can be put into a single condition a la

```
if ( newContractsConfig.contracts[contractName].gas && newContractsConfig.contracts[contractName].gas != 'auto')
```

Some notes on this:

- We should get rid off the magic 'auto' string as mentioned above.
- It prolly makes sense to put `gas` in a separate variable so the condition doesn't get so utterly long